examples/language-modeling/fsdp_config.json

```diff
@@ -8,5 +8,6 @@
     "fsdp_sync_module_states": true,
     "fsdp_use_orig_params": true,
     "transformer_layer_cls_to_wrap": "GaudiLlamaDecoderLayer",
-    "fsdp_activation_checkpointing": false
```

Comment:
Please also update peft version in requirement file (ignore if it is already done)

Reply:
For PEFT, the current version is actually 0.12.0 from the docker itself. 
Single pytest, and diffuser test install peft to 0.10.0 and 0.7.0 for their test, but default is 1.12.0 and also in language-modeling requirements also specify peft as 0.12.0 so I think we are ok.